lib/models/migration.js

```diff
@@ -45,6 +45,51 @@ module.exports = function(Migration, options) {
   };
 
   /**
+   * Remote Method: Run specific migration by name.
+   *
+   * @param {String} [name] Name of migration script to run.
+   * @param {String} [record] Record the migration runtime to database.
+   * @param {Function} [cb] Callback function.
+   */
+  Migration.migrateByName = function(name, record, cb) {
+    if (typeof cb === 'undefined' && typeof record === 'function') {
+      cb = record;
+      record = false;
+    }
+
+    record = record || false;
+    cb = cb || utils.createPromiseCallback();
+    assert(typeof name === 'string', 'The to argument must be a string, not ' + typeof name);
+    assert(typeof cb === 'function', 'The cb argument must be a function, not ' + typeof cb);
+
+    Migration.log.info(name, 'running.');
+    const scriptPath = path.resolve(path.join(Migration.migrationsDir, name));
+
+    try {
+      require(scriptPath).up(Migration.app, function(err) {
+        if (err) {
+          Migration.log.error(`Error running migration script ${name}:`, err);
+          Migration.finish(err);
+          return cb(err);
+        } else if (record) {
+          Migration.create({
+            name: name,
+            runDtTm: new Date()
+          });
+          cb();
```

Comment:
I think this needs to move to below this if block since we want to call it regardless of the value of  `record`